src/backend/marsha/core/tests/api/live_sessions/test_create.py

```diff
@@ -34,9 +34,15 @@
 class LiveSessionCreateApiTest(LiveSessionApiTestCase):
     """Test the create API of the liveSession object."""
 
+    def _post_url(self, video):
+        """Return the url to use to create a live session."""
+        return f"/api/videos/{video.pk}/livesessions/"
```

Comment:
I don't think we should add this abstraction.
IMO, we should explicitly see the called url in the tests

Reply:
This abstraction is for the sake of testing both old endpoint and new ones with the same code. With the suppression of the old endpoints, this can be revert to normal.

Reply:
Oh ok, didn't see it clearly.